### PR TITLE
[tests] Remove unneeded helper function. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: tests/test_core.py [hunk 1]
```diff
@@ -3301,7 +3284,6 @@ def test_dlfcn_longjmp(self):
         printf("pre %d\n", i);
       }
       ''')
-    self.set_setting('EXPORTED_FUNCTIONS', ['_jumpy'])
```
Comment (kripken) [inline tests/test_core.py:3304]: We'd need this if we used mode 2, I think? Looks like we don't use that for these tests, so this should be an ok change, but maybe it's worth keeping for future use if we do want mode 2 here?

File: tests/test_core.py [hunk 2]
```diff
@@ -3365,7 +3346,6 @@ def zzztest_dlfcn_exceptions(self):
       }
       }
       ''')
-    self.set_setting('EXPORTED_FUNCTIONS', ['_ok', '_fail'])
```
Comment (kripken) [inline tests/test_core.py:3368]: (same)

**Review by kripken — APPROVED** [review]

File: tests/test_core.py (hunk 1 above)
Comment (sbc100) [reply tests/test_core.py:3304]: Sure.. if we ever wanted to use `SIDE_MODULE=2` then we would need `EXPORTED_FUNCTIONS`.  As it stands I think this does nothing..

Using `SIDE_MODULE=2` would imply there is stuff in the side module that we don't need.. which is almost never the case in test code.

Comment (sbc100) [reply tests/test_core.py:3304]: I'd rather not keep settings around that are not used.

**Pull request merged**